portality/view/api_v1.py

```diff
@@ -40,61 +40,322 @@ def list_operations():
+                      author_pays:
+                        type: string
+                      license:
+                        type: array
+                        items:
+                          type: object
+                          title: license
+                          properties:
+                            title:
+                              type: string
+                            url:
+                              type: string
+                            NC:
+                              type: boolean
+                            ND:
+                              type: boolean
+                            embedded_example_url:
+                              type: string
+                            SA:
+                              type: boolean
+                            type:
+                              type: string
+                            BY:
+                              type: boolean
+                      title:
+                        type: string
+                      author_pays_url:
+                        type: string
+                      country:
+                        type: string
+                      link:
+                        type: array
+                        items:
+                          type: object
+                          title: link
+                          properties:
+                            url:
+                              type: string
+                            type:
+                              type: string
+                      provider:
+                        type: string
+                      keywords:
+                        type: array
+                        items:
+                            type: string
+                      oa_start:
+                        type: object
+                      identifier:
+                        type: array
+                        items:
+                          type: object
+                          title: identifier
+                          properties:
+                            type:
+                              type: string
+                            id:
+                              type: string
+                      subject:
+                        type: array
+                        items:
+                          type: object
+                          title: subject
+                          properties:
+                            code:
+                              type: string
+                            terms:
+                              type: string
+                            scheme:
+                              type: string
+                  created_date:
+                    type: string
+                    format: dateTime
+            query:
+              type: string
+            total:
+              type: integer
+            page:
+              type: integer
+      400:
+        description: Bad Request
+    """
```

Comment:
A bit long-ish but good I think. I'm sure we can move off the docstrings to someplace else if really necessary.